.github/workflows/shared.yml

```diff
@@ -135,6 +135,7 @@ jobs:
         if: ${{ matrix.target == 'linux-x86_64' && inputs.fake_tests }}
         run: |
           # See https://github.com/urbit/vere/issues/40.
+          bazel build :urbit --config=mem_dbg
```

Comment:
We probably shouldn't use the `--config=` config definitions defined in .bazelrc in ci. These are only meant for local development usage. Much more foolproof and easier to reason about would be the following:

`bazel build --copt='-DU3_CPU_DEBUG` --copt='-DU3_MEMORY_DEBUG' :urbit`

Reply:
Actually, I believe all of these options are already getting invoked. https://github.com/urbit/vere/blob/7ef74854aa5c0d68a155e113299e8f1669f73a96/.bazelrc#L34

Reply:
Right, so maybe all we need to do is add the `--gc` flag to the `urbit` flags we use for the fake ship tests.